languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/hunspell/create_dict.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+echo "Create morfologik spelling dictionary, based on Hunspell dictionary"
```

Comment:
we use now a cleaned version ([see](https://github.com/sohaibafifi/languagetool/blob/arabic-dev/languagetool-language-modules/ar/src/main/resources/org/languagetool/resource/ar/hunspell/create_dict.sh)) but it doesn't work ! please see [the log ](https://travis-ci.org/sohaibafifi/languagetool/builds/220696836 )